library/src/main/java/com/northconcepts/templatemaster/form/CrudResource.java

```diff
@@ -167,7 +167,7 @@ public Content getListRecords(@QueryParam("q") String searchQuery, @QueryParam("
     @Path("/{pageNumber}")
     public Content getListRecords(@QueryParam("q") String searchQuery, @QueryParam("s")String sortField, @PathParam("pageNumber") int pageNumber) {
         searchQuery =  Util.isNotEmpty(searchQuery)?searchQuery:null;
-        sortField = Util.isNotEmpty(sortField)?sortField:null;
+        sortField = Util.isNotEmpty(sortField)?sortField:formDef.getDefaultSortField();
```

Comment:
I'll wait this to be updated as per our discussion.

Reply:
yes, setting default sort field is not required.